### PR TITLE
Adds a new business use case scenario

### PR DESCRIPTION
**Pull request by carolinaborim**
This commit includes a requested business use case around how to access
equipment real time data.

@carolinaborim & @glauco pairing

### CODE REVIEW

File: source/includes/_canVariables.md [hunk 1]
```diff
@@ -1,6 +1,74 @@
 # CAN Variables
 
-## Description
+## Getting CAN variable values
+
+<blockquote class='lang-specific curl'>
+<p>1. List all equipment you have access using the following command:</p>
```
Comment (waldemarnt) [inline source/includes/_canVariables.md]: What about change the personal context to be generic? I mean s/all equipment you have access/all available equipment for user/ something like that. In my point of view this part of the documentation is not an use case directly, and have to follow the same context of the others API resources. What do you think?

File: source/includes/use_case_scenarios/_accessing_equipment_near_real_time_data.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    * Check out [how to do it using our API]()
+* Measure performance based on fuel consumption and DEF consumption on an
+instant, average, and total basis
+    * Check out [how to do it using our API]()
+  depending on machine type, know:
+    * The bin/tank level, estimated time/area until empty/full
+        * Check out [how to do it using our API]()
+    * Additional performance information like area/hour, speed, yield or
+  application/seeding rate
+        * Check out [how to do it using our API]()
```
Comment (waldemarnt) [inline source/includes/use_case_scenarios/_accessing_equipment_near_real_time_data.md]: As I understood, from the line 18 to 27 uses the same documentation url right? What about merge this in the same context? Like that:
"Using the can variables documentation you are able to cover this common scenarios:
- Know the last available fuel/DEF level of the machine, its location and heading
- Measure performance based on fuel consumption and DEF consumption on an instant, average, and total basis
..."
Make sense?

File: source/includes/_canVariables.md (hunk 1 above)
Comment (glauco) [inline source/includes/_canVariables.md]: It certainly makes sense!
I already updated it to reflect your feedback.

File: source/includes/use_case_scenarios/_accessing_equipment_near_real_time_data.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+section](#can-variables). The amount of available data will depend on the
+contracted data collection plan (standard+ or advanced).
+
+Equipment information will be available to you, or any 3rd party system you
+contract, through our APIs. Our goal is to help you managing your entire
+operation.
+
+Here are a few examples of what you could do with the data we currently offer:
+
+* Know status (working, idle, etc.) for all my machines to manage my entire farm
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_accessing_equipment_near_real_time_data.md]: Maybe make this a little bit more impersonal? For example:
s/Know status (working, idle, etc.) for all my machines to manage my entire farm operation/Get current status (working, idle, etc.) for all machines in order to assist managing your entire farm operation./

File: source/includes/use_case_scenarios/_listing_models.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+1. The first step is to get a list of [brands](#get-brands) resource on `Telemetry API`.
+
+2. The next request will filter [series](#series-request-parameters) linking them with a `brand` using the
+brand identifier returned from the previous one.
+
+3. Given that the previous requests caught a list of `series` that was filtered by `brands` this step
+has to list `equipment types`. The [equipment types](#get-equipmenttypes) documentation shows how to get
+a list of them using the HTTP method `GET`.
+
+4. The last step of the flow has to list models of an `equipment type` and `serie`. Using the `series`
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_listing_models.md]: s/serie/series

File: source/includes/use_case_scenarios/_listing_models.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+identifier and `equipment types` identifier is it possible to make a request to [models](#models-request-parameters)
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_listing_models.md]: s/is it/it is

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 6]
```diff
@@ -3,136 +3,11 @@
...
+* [Accessing equipment near real time
+  data](#accessing-equipment-near-real-time-data)
+* [Listing models filtering by series, types and
+  brands](#listing-models-filtering-by-series-types-and-brands)
+
 ## Credentials
 In the following samples a valid Bearer
 Token in required in the `Authorization Header` of every HTTP Request to the API.
-
-If you don't have a valid Bearer Token, refer to this [section](https://agco-fuse.github.io/documentation/#authenticate-using-oauth2).
```
Comment (lucaslago) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: Was this paragraph's removal intended? I think a link to the authentication section of our docs makes sense here.

File: source/includes/use_case_scenarios/_accessing_equipment_near_real_time_data.md (hunk 2 above)
Comment (glauco) [inline source/includes/use_case_scenarios/_accessing_equipment_near_real_time_data.md]: We updated it making these links inline. Does that work?

**Comment by waldemarnt** [comment]
👍

**Comment by lucaslago** [comment]
👍

**Pull request merged**